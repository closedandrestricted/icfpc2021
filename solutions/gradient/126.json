{"vertices":[[86,168],[50,201],[87,218],[60,234],[70,229],[92,205],[123,182],[67,201],[36,241],[74,258],[85,197],[92,204],[92,179],[81,229],[54,200],[39,204],[83,194],[64,212],[43,264],[60,187],[70,214],[56,234],[57,259],[74,232],[70,199],[52,226],[65,237],[55,208],[48,216],[39,220],[46,232],[50,237],[84,194],[79,247],[75,241],[82,224],[55,199],[42,212],[40,215],[48,195],[61,179],[67,216],[37,190],[49,252],[53,222],[63,202],[55,220],[72,214],[75,214],[72,203],[75,242],[87,183],[64,201],[25,228],[52,206],[101,237],[30,249],[80,213],[62,246],[72,223],[77,242],[55,233],[76,234],[32,199],[55,223],[98,225],[102,229],[84,259],[61,195],[134,227],[61,230],[65,211],[74,214],[57,223],[55,221],[46,223],[72,197],[56,233],[46,213],[65,192],[76,242],[55,216],[82,215],[55,180],[86,237],[85,212],[65,223],[84,212],[78,210],[77,229],[68,235],[46,219],[61,234],[79,216],[53,230],[103,214],[74,219],[48,193],[73,211],[105,247],[55,200],[55,220],[63,214],[56,222],[71,213],[86,186],[50,230],[47,222],[65,218],[83,234],[56,244],[57,194],[44,214],[46,221],[62,239],[68,204],[54,198],[82,234],[74,212],[57,198],[63,243],[89,219],[49,254],[47,177],[49,196],[46,230],[66,176],[76,211],[57,191],[59,180],[72,189],[81,238],[74,221],[60,195],[64,220],[80,250],[93,243],[79,222]]}