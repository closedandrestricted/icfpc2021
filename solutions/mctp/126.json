{"vertices":[[58,281],[14,286],[0,248],[9,223],[51,235],[36,250],[20,287],[26,252],[49,199],[0,249],[17,228],[1,252],[39,252],[9,278],[12,220],[15,218],[27,246],[13,254],[10,279],[38,245],[1,229],[0,253],[21,249],[49,214],[5,230],[29,252],[0,256],[25,249],[0,258],[0,231],[0,250],[1,259],[16,233],[27,251],[1,233],[0,258],[1,257],[13,224],[14,226],[1,238],[4,258],[2,231],[0,230],[1,228],[1,229],[24,243],[23,240],[19,254],[41,203],[25,244],[30,228],[3,231],[1,259],[22,254],[9,246],[0,257],[10,258],[0,254],[2,255],[25,267],[32,229],[0,230],[0,232],[0,265],[2,231],[48,251],[0,240],[27,256],[22,251],[3,271],[0,229],[23,230],[24,239],[1,233],[0,230],[1,237],[0,259],[31,236],[10,244],[36,256],[19,252],[26,228],[0,264],[1,235],[9,259],[28,221],[2,256],[33,225],[16,250],[1,237],[18,254],[5,269],[2,250],[2,231],[16,257],[36,265],[13,271],[0,263],[3,231],[40,233],[29,270],[0,233],[3,270],[41,248],[24,281],[16,239],[0,249],[27,248],[1,266],[15,275],[0,243],[28,258],[21,275],[27,247],[1,259],[2,252],[16,257],[24,242],[7,258],[41,270],[0,247],[12,267],[12,265],[9,257],[24,257],[34,238],[17,274],[0,259],[0,264],[23,266],[27,281],[0,232],[0,247],[29,255],[3,252],[0,237],[6,275],[2,227]]}