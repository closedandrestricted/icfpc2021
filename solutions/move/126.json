{"vertices":[[64,175],[45,240],[83,223],[64,238],[54,240],[75,212],[102,183],[49,208],[18,247],[67,265],[72,218],[74,210],[72,185],[63,235],[36,207],[26,210],[66,202],[46,218],[35,268],[58,195],[55,224],[38,242],[38,265],[57,239],[52,205],[34,233],[53,241],[40,212],[31,224],[21,228],[39,237],[32,243],[74,215],[63,252],[57,247],[64,230],[45,217],[24,222],[24,226],[30,201],[43,185],[50,222],[21,197],[42,225],[35,229],[47,207],[37,227],[54,220],[58,220],[54,209],[59,248],[69,189],[47,207],[8,228],[41,207],[83,243],[9,246],[62,219],[46,253],[55,229],[61,248],[39,247],[58,240],[33,226],[38,230],[81,230],[88,214],[85,216],[44,202],[117,226],[43,236],[47,217],[56,220],[48,224],[60,232],[28,230],[61,203],[38,239],[29,218],[49,198],[59,248],[37,223],[65,222],[38,197],[69,242],[68,220],[47,229],[67,218],[60,216],[60,235],[50,241],[36,228],[49,240],[61,224],[35,236],[85,219],[56,225],[33,197],[55,217],[88,251],[38,208],[38,228],[45,220],[39,230],[53,219],[68,192],[32,236],[29,230],[48,224],[66,240],[45,253],[40,200],[26,220],[29,229],[46,248],[50,210],[36,204],[65,239],[56,218],[39,204],[46,250],[73,227],[32,262],[29,184],[31,202],[28,238],[48,182],[60,217],[39,198],[41,186],[54,195],[62,244],[56,224],[40,202],[45,227],[63,258],[75,245],[61,228]]}