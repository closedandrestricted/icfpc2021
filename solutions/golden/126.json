{"vertices":[[72,167],[52,233],[90,216],[71,231],[61,233],[82,205],[109,176],[56,201],[25,240],[74,258],[79,211],[81,203],[79,178],[70,228],[43,200],[33,203],[73,195],[53,211],[43,262],[65,188],[62,217],[45,235],[45,258],[64,232],[59,198],[41,226],[60,234],[47,205],[38,217],[28,221],[46,230],[39,236],[81,208],[70,245],[64,240],[71,223],[52,210],[31,215],[31,219],[37,194],[50,178],[57,215],[27,191],[49,218],[42,222],[54,200],[44,220],[61,213],[65,213],[61,202],[66,241],[76,182],[54,200],[15,226],[48,200],[90,236],[17,244],[69,212],[53,246],[62,222],[68,241],[46,240],[65,233],[40,219],[45,223],[88,223],[95,207],[92,209],[51,195],[124,221],[50,229],[54,210],[63,213],[55,217],[67,225],[35,223],[68,196],[45,232],[36,211],[56,191],[66,241],[44,216],[72,215],[45,190],[76,235],[75,213],[54,222],[74,211],[67,209],[67,228],[57,234],[43,221],[56,233],[68,217],[42,229],[92,212],[63,218],[40,190],[62,210],[95,244],[45,201],[45,221],[52,213],[46,223],[60,212],[75,185],[39,229],[36,223],[55,217],[73,233],[52,246],[47,193],[33,213],[36,222],[53,241],[57,203],[43,197],[72,232],[63,211],[46,197],[53,243],[80,220],[39,255],[36,177],[38,195],[35,231],[55,175],[67,210],[46,191],[48,179],[61,188],[69,237],[63,217],[47,195],[52,220],[70,251],[82,238],[68,221]]}